src/test/java/com/artipie/asto/ext/ContentDigestTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.asto.ext;
+
+import com.artipie.asto.Content;
+import java.security.MessageDigest;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link ContentDigest}.
+ *
+ * @since 0.6
+ */
+final class ContentDigestTest {
+
+    @Test
+    void calculatesHex() throws Exception {
+        final MessageDigest sha = MessageDigest.getInstance("SHA-256");
+        MatcherAssert.assertThat(
+            new ContentDigest(
+                new Content.From(
+                    // @checkstyle MagicNumberCheck (1 line)
+                    new byte[]{(byte) 0xca, (byte) 0xfe, (byte) 0xba, (byte) 0xbe}
+                ),
+                () -> sha
+            ).hex().toCompletableFuture().get(),
+            new IsEqual<>("65ab12a8ff3263fbc257e5ddf0aa563c64573d0bab1f1115b9b107834cfa6971")
```

Comment:
@g4s8 I'd suggest to use `DigestUtils.sha256Hex` from apache-commons here

Reply:
@olenagerasimova I don't really like expected test value runtime evaluation, I prefer fixed constants to make tests more robust. When I wrote this test, I calculated this hex string using Python interpreter and copy-paste it here.